sdk/src/Core/Amazon.Runtime/Credentials/InstanceProfileAWSCredentials.cs

```diff
@@ -180,14 +181,19 @@ public InstanceProfileAWSCredentials(string role, IWebProxy proxy)
         }
 
         /// <summary>
-        /// Constructs a InstanceProfileAWSCredentials object for the first found role
+        /// Constructs <see cref="InstanceProfileAWSCredentials"/> for the first found role.
         /// </summary>
+        /// <remarks>This retrieves the name of the IAM role associated with the instance during construction.</remarks>
```

Comment:
I know that it is implied that this makes an IMDS call, but should we be more explicit and state that it does make an IMDS call? Something like:

`This makes an IMDS call to retrieve the name of the IAM role associated with the instance during construction`

Reply:
Addressed via force push.